Task3.py

```diff
@@ -0,0 +1,30 @@
+# Multi-paradigm programming languages, Task №3
+# Andrey Fortus IKM-221a
+import math
+
+GENERAL_INFO = 'Multi-paradigm programming languages, Task №3'
+STUDENT_INFO = 'Andrey Fortus IKM-221a'
+TEMPLATE = 'Input {} value: '
+
+# info
+print(GENERAL_INFO)
+print(STUDENT_INFO)
+
+# Math
+while True:
+    try:
+        x = float(input(TEMPLATE.format('x')))
+        z = float(input(TEMPLATE.format('z')))
+        if z - pow(x, 3) / 3 == 0:
```

Comment:
look, you have a few validations to avoid division by zero.
but the validations do nothing. you just print some message and then proceed to division by zero.
rewrite your code in a manner where you don't need the second try catch.